packages/react-native-web/src/exports/Image/index.js

```diff
@@ -146,6 +147,23 @@ function resolveAssetUri(source): ?string {
   return uri;
 }
 
+function raiseOnErrorEvent(uri, { onError, onLoadEnd }) {
+  if (onError) {
+    onError({
+      nativeEvent: {
+        error: `Failed to load resource ${uri} (404)`
+      }
+    });
+  }
+  if (onLoadEnd) onLoadEnd();
+}
```

Comment:
This was extracted for reuse out of the original Image component hook
(We reuse this in case loading with headers fails)